docs/user/tutorials/01-10-external-authorization-provider.md

```diff
@@ -0,0 +1,175 @@
+# External Authorization Provider
+
+This tutorial shows how to expose and secure an HTTPBin Service using an external authorization provider.
+
+## Prerequisites
+
+* Kyma installation with the Istio module enabled. If you use a Kyma domain, also the API Gateway module must be enabled.
+* [Deploy a sample HTTPBin Service](https://kyma-project.io/#/api-gateway/user/tutorials/01-00-create-workload).
+* Set up [your custom domain](https://kyma-project.io/#/api-gateway/user/tutorials/01-10-setup-custom-domain-for-workload) or use a Kyma domain instead.
+* Depending on whether you use your custom domain or a Kyma domain, export the necessary values as environment variables:
+
+    <!-- tabs:start -->
+    #### **Custom Domain**
+
+    ```bash
+    export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+    export GATEWAY=$NAMESPACE/httpbin-gateway
+    ```
+    #### **Kyma Domain**
+
+    ```bash
+    export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
+    export GATEWAY=kyma-system/kyma-gateway
+    ```
+    <!-- tabs:end -->
+
+## Steps
+
+### Expose an HTTPBin Service
+
+Create a VirtualService to expose the HTTPBin workload.
+
+    ```shell
```

Comment:
As far as I can see shell is not a [valid alias](https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L6475). We also don't use an alias for the other code blocks, is there a reason why we use it only here?
```suggestion
    ```bash
```